packages/themeables/spacer/changelog.md

```diff
@@ -0,0 +1,13 @@
+## v1.0.1
```

Comment:
this file should be removed, looks like an artifact from copying an old package.